### ISSUE TITLE
[FR] Figure component should support alt text

### ISSUE THREAD

**Issue by tomerlichtash** [tomerlichtash/mels-loop#346]
Currently, image component parses only the image `src`. It should be able to parse the alt text attached to the image as well, like so:

```
<figure>
![some alt text]link-to-image.jpg)
<figcaption>some figcaption</figcaption>
</figure>
```
